extensions/wikia/ThemeDesigner/js/ThemeDesigner.js

```diff
@@ -770,7 +744,12 @@
 		themes: false,
 
 		applySettings: function (reloadCSS, updateSkinPreview) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has a complexity of 11 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)